src/agent/at-driver.js

```diff
@@ -32,7 +32,11 @@ export class ATDriver {
     this.socket = socket;
     this.log = log;
     this.ready = new Promise(resolve => socket.once('open', () => resolve())).then(() =>
-      this._send({ method: 'session.new', params: { capabilities: {} } })
+      this._send({ method: 'session.new', params: { capabilities: {} } }).then(
+        ({ result: { capabilities } }) => {
+          this.capabilities = capabilities;
```

Comment:
Let's store this as `_capabilities` and expose an async `getCapabilities` method. That'll increase parity with the `WebDriver` class and also make the call site a little cleaner.

Reply:
The `WebDriver` `getCapabilities()` ends up giving back an object that then has `get(keyName)` method, would you want me to kind of mirror that same API, or just return the whole object at that point?

```js
  async getCapabilities() {
    const capabilities = await this.webDriver.getCapabilities();
    const browserName = capabilities.get('browserName');
    const browserVersion = capabilities.get('browserVersion');
    await this.atDriver.ready;
    const { atName, atVersion, platformName } = this.atDriver.capabilities;
    return { atName, atVersion, browserName, browserVersion, platformName };
  }
```

note the `browserName` and `browserVersion` = `capabilities.get()` calls

The current at-driver we are using (PAC's) does not support getting the capabilities back at any time other than the response to the `session.new`, so we still need to store it here on the connection for now.